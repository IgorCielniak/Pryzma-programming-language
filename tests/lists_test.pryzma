a = [1,2,3]


append a, "text"

print a,"\n"

pop a, 2

print a

