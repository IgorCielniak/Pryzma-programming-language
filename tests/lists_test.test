python Pryzma.py tests/lists_test.pryzma
