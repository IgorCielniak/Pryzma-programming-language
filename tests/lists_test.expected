[1, 2, 3, 'text']
[1, 2, 'text']
